### DISCUSSION TITLE
How to use monai do extract high level features of 2D Image

### DISCUSSION THREAD

**Discussion by lcgmu** [Project-MONAI/tutorials#1073] [Ideas]
Hi, I'm a beginner of pytorch and monai，I have a question about how to extract the high level features of 2D image (eg：after multil layers convolutions), for example , if I use resnet to classification, but I only need  the data of the  layer befor FC（Full connect）, should  manual edit the resnet or have other method to resolve my problems. thanks for answer my question.

**Comment by wyli** [comment]
I think you can try the feature_extraction from torchvision 
 https://pytorch.org/vision/stable/feature_extraction.html
